docs/tutorials/optimize/power/power_manage.md

```diff
@@ -1,3 +1,271 @@
-# Power Optimization
+# Power optimization
 
-[A document about power management]
+IoT devices often claim they can run ten years on one battery, but building low-powered nodes is challenging. Complex products use multiple threads, a variety of active timers and sometimes a second core handling network connectivity. Sensors and other peripherals also use power. To help you manage these complexities while using the least amount of power, Mbed OS contains a number of low-power features, including low power tickers, tickless mode and the sleep manager. This tutorial explains how to use these features.
+
+## Power modes
+
+Mbed OS contains three power modes:
+
+- Active - The MCU and all clocks are running.
+- Sleep - The core system clock is disabled. This eliminates dynamic power that the processor, memory systems and buses use.
+- Deep sleep - In addition to the core system clock, all high-frequency clocks are disabled, and the [SysTick](../apis/rtos.html) is disabled.
+
+Switching between these power modes occurs automatically. When all threads in the system are idle, Mbed OS yields control to the [idle thread](../apis/idle-loop.html). The idle thread then invokes the sleep manager, which brings the system to sleep or deep sleep mode. The idle thread also sets a timer to wake up the system again, but you can also wake up the system through an external interrupt or the Real-Time Clock (RTC).
```

Comment:
Mbed OS doesn't support waking of the RTC, can we replace it with low power ticker.